content/_index.html

```diff
@@ -0,0 +1,407 @@
+          <a href="/learn-more/#use-cases">Find out more from RAPIDS Use Cases <i class="fa-solid fa-arrow-up-right"></i> </a>
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-light fa-chart-network"></i> Open Source Ecosystem </h2>
+          <p>RAPIDS is Open Source and available on <a href="https://github.com/rapidsai" taget="_blank">GitHub</a>. Our mission is to empower and advance the open-source GPU data science ecosystem and work toward flawless integration. </p>
+          <a href="https://github.com/rapidsai">Jump to RAPIDS on GitHub <i class="fa-solid fa-arrow-up-right"></i> </a>
+        </div>
+      </div>
+
+      <div class="columns mt-5">
+        <div class="column is-one-third">
+          <h2><i class="fa-thin fa-objects-column"></i> Faster Pandas <br> with cuDF</h2>
+          <p>cuDF is nearly a drop in replacement to Pandas for most use cases and has greatly improved performance.</p>
+
+          <canvas id="cuDFchart" width="400" height="200"></canvas>
+          <p>*A100 vs i7-6000</p>
+          <a href="#">Try out this notebook benchmark <i class="fa-solid fa-arrow-up-right"></i> </a>
+
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-light fa-microchip-ai"></i> Faster Scikit-learn <br> with cuML</h2>
+          <p>cuML is nearly a drop in replacement to scikit-learn for most use cases and has greatly improved performance.</p>
+          
+          <canvas id="cuMLchart" width="400" height="200"></canvas>
+          <p>*A100 vs i7-6000</p>
+          <a href="#">Try out this notebook benchmark <i class="fa-solid fa-arrow-up-right"></i> </a>
+
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-light fa-circle-nodes"></i> Faster networkX <br> with cuGraph</h2>
+          <p>cuGraph is nearly a drop in replacement to networkX for most use cases and has greatly improved performance.</p>
+
+          <canvas id="cuGraphChart" width="400" height="200"></canvas>
+          <p>*A100 vs i7-6000</p>
+          <a href="#">Try out this notebook benchmark <i class="fa-solid fa-arrow-up-right"></i> </a>
+
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <div class="top-slant-container">
+    <div class="top-slant-down has-background-rapids-purple-primary"></div>
+  </div>
+
+  <!-- Quick Start -->
+  <div id="quick-start"></div>
+  <section class="section has-background-rapids-purple-primary has-text-white padding-bottom-slant">
+    <div class="container">
+      <h1 class="mb-5 has-text-white">Quick Start</h1>
+      <div class="columns">
+        <div class="column is-half">
+          <h2 class="has-text-white"><i class="fa-light fa-download"></i> Quick Local Install </h2>
+          <p>RAPIDS offers several installation methods, the quickest is shown below. For more detailed guides go to our <a href="https://docs.rapids.ai/install">install page <i class="fa-solid fa-arrow-up-right"></i></a> </p>
```

Comment:
Should we call this the "advanced install" page? we call it that lower on this page and I think the name fits.